relacy/base.hpp

```diff
@@ -110,10 +112,10 @@ T val(T x)
 
 template <class From, class To, template <class> class CheckTrait, template <class> class AddTrait>
 struct copy_trait_impl {
-    using type = std::conditional_t<
+    using type = typename std::conditional<

```

Comment:
Updated this to use C++11 compatible `std::conditional`, as there's no strong reason to *force* C++17 or newer as a requirement.